src/Stan/Report/Html.hs

```diff
@@ -41,334 +40,318 @@ import Stan.Severity (Severity (..), severityDescription)
+stanSeverityExplained :: Html
+stanSeverityExplained = do
+    divClass "col-5" $
+        blockP "We are using the following severity system to indicate the observation level"
 
-    # tableWithShadow "col-7"
-        ( tr_A greyBg (th_ "Severity" # th_ "Description")
-        # map toSeverityRow (universe @Severity)
-        )
+    tableWithShadow "col-7" $ do
+        tr ! greyBg $ (th "Severity" >> th "Description")
+        sequence_ $ map toSeverityRow (universe @Severity)
   where
-    toSeverityRow s = tr_
-        ( td_ (severity $ show @Text s)
-        # td_ (severityDescription s)
-        )
-
-stanFooter = footer_
-    ( divClass "container"
-        ( divClass "row footer-link"
-            ( span_ "This report was generated by "
-            # a_A (A.href_ "https://github.com/kowainik/stan") "Stan — Haskell Static Analysis Tool."
-            )
-        # divClass "row footer-link"
-            ( span_ "Stan is created and maintained by "
-            # a_A (A.href_ "https://kowainik.github.io") "Kowainik"
-            )
-        )
-    # nav_A (A.class_ "row centre") (h3_ $ strong_ "© Kowainik 2020")
-    )
-
-stanHead = head_
-    ( meta_A (A.httpEquiv_ "Content-Type" # A.content_ "text/html; charset=UTF-8")
-    # meta_A (A.httpEquiv_ "X-UA-Compatible" # A.content_ "IE=Edge")
-    # nameContent "viewport" "width=device-width, initial-scale=1.0"
-    # nameContent "description" "Stan Report"
-    # nameContent "keywords" "Haskell, Static Analysis"
-    # nameContent "author" "Kowainik"
-    # title_ "Stan Report"
-
-    # style_ (Raw $ renderWith compact [] stanCss)
-    )
+    toSeverityRow :: Severity -> Html
+    toSeverityRow s = tr $ do
+        td (severity $ show s)
+        td (toHtml $ severityDescription s)
+
+stanFooter :: Html
+stanFooter = footer $ do
+    divClass "container" $ do
+        divClass "row footer-link" $ do
+            span "This report was generated by "
+            a ! (A.href "https://github.com/kowainik/stan") $
+                toHtml @Text "Stan — Haskell Static Analysis Tool."
+        divClass "row footer-link" $ do
+            span "Stan is created and maintained by "
+            a ! (A.href "https://kowainik.github.io") $ toHtml @Text "Kowainik"
+    nav ! (A.class_ "row centre") $ h3 $ strong "© Kowainik 2020"
+
+stanHead :: Html
+stanHead = head $ do
+    meta ! (A.httpEquiv "Content-Type" <> A.content "text/html; charset=UTF-8")
+    meta ! (A.httpEquiv "X-UA-Compatible" <> A.content "IE=Edge")
+    nameContent "viewport" "width=device-width, initial-scale=1.0"
+    nameContent "description" "Stan Report"
+    nameContent "keywords" "Haskell, Static Analysis"
+    nameContent "author" "Kowainik"
+    title "Stan Report"
+
+    style (toHtml $ renderWith compact [] stanCss)
   where
-    nameContent x y = meta_A (A.name_ x # A.content_ y)
+    nameContent x y = meta ! (A.name x <> A.content y)
 
-stanJs = script_ $ Raw $ List.unlines
+stanJs = script $ toHtml $ List.unlines
```

Comment:
Should add top-level type signature here as well and remove `OPTIONS_GHC` from this module?